src/modules/gnd_pos_control/GroundRoverPositionControl.hpp

```diff
@@ -133,7 +126,6 @@ class GroundRoverPositionControl
 	struct fw_pos_ctrl_status_s			_gnd_pos_ctrl_status;		/**< navigation capabilities */
 	struct manual_control_setpoint_s		_manual;			/**< r/c channel data */
 	struct vehicle_control_mode_s			_control_mode;			/**< control mode */
-	struct vehicle_command_s			_vehicle_command;		/**< vehicle commands */
 	struct vehicle_status_s				_vehicle_status;		/**< vehicle status */
 	struct vehicle_land_detected_s			_vehicle_land_detected;		/**< vehicle land detected */
```

Comment:
Land detection is gone now. We should probably do some testing before more hacking, but the only missing piece is likely resetting the integrators when not moving.